lxd/instance/drivers/driver_qemu.go

```diff
@@ -4964,3 +4964,47 @@ func (d *qemu) writeInstanceData() error {
 
 	return nil
 }
+
+// Info returns "qemu" and the currently loaded qemu version.
+func (d *qemu) Info() instance.Info {
+	var qemuVersion, qemuBinary, qemuPath string
+	var out []byte
+	var err error
```

Comment:
Unless absolutely needed, we usually prefer defining variables as we go using `:=`.

Reply:
Fixed in 37c1ac5 - that was a remnant of an old version of the code that _did_ require some of those to be defined at the top, and is no longer necessary.